website/src/content/help-needed.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Help needed?
+---
+
+<Subtitle>
+
+In this page we will document some things that might help clarify certain topics or issue.
+
+</Subtitle>
+
+<Info>
+
+Something is not working and you need help? Do you have a technical question, or are you simply looking for some advice?
```

Comment:
```suggestion
Something is not working as expected? You need general guidance? Do you have a technical question or are simply looking for some advice?
```

Reply:
```suggestion
Something is not working as expected? Do you need general guidance? Do you have a technical question or are you simply looking for some advice?
```

PS: 
<img width="1390" alt="image" src="https://user-images.githubusercontent.com/1110551/76979480-467e7880-6938-11ea-8187-0b484cc04fc3.png">

Reply:
Sounds good to me.